File: src/Data/Order.java
package Data;

public class Order {
	private long product;

	public Order(long product) {
		this.product = product;
	}

	Order() {

	}

	private long calcCost() {
		return (long) (getPCost() * getProduct());
	}

	private long calcGCT() {
		return (long) (calcCost() * getGCT());
	}

	private double getGCT() {
		return 0.165;
	}

	public long total() {
		return (calcCost() + calcGCT());
	}


	private long getProduct() {
		return product;
	}

	private float getPCost() {
		return 600;
	}

	public String toString() {
		return "Orders amount: " + "$" + this.total() + OrderStatus.findByName("Active");
	}

}

File: src/data/Order.kt
package data

class Order(private var product: Long) {

    private val gct: Double
        get() = 0.165

    private val pCost: Float
        get() = 600f

    private fun calcCost(): Long {
        return (pCost * product).toLong()
    }

    private fun calcGCT(): Long {
        return (calcCost() * gct).toLong()
    }

    fun total(): Long {
        return calcCost() + calcGCT()
    }

    override fun toString(): String {
        return "Orders amount: " + "$" + this.total() + OrderStatus.findByName("Active")
    }

}
